pkg/operator/eventwatch/controller.go

```diff
@@ -99,7 +99,7 @@ func (c *Controller) getEventHandler(eventKey string) *eventHandler {
 		return nil
 	}
 	for i := range c.events {
-		if c.events[i].namespace == namespace && c.events[i].reason == reason {
+		if c.events[i].namespace == namespace && strings.Contains(reason, c.events[i].reason) {
```

Comment:
this means `reason` is actually `reasonSubstring` everywhere? Would prefer to make that explicit rather than introducing magic behaviour here.